src/Spd.Manager.Licence/SharedContract.cs

```diff
@@ -89,7 +89,9 @@ public enum PoliceOfficerRoleCode
     CourtAppointedBailiff,
     SpecialProvincialOrMunicipalConstable,
     PoliceOfficer,
+    PoliceOfficerRetired,
     Other,
+    None,

```

Comment:
'None' should not be visible to the frontend. Please remove.

Reply:
removed